contracts/AcceleratingDistributor.sol

```diff
@@ -150,17 +150,22 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
     }
 
     /**
-     * @notice Recover an ERC20 token either dropped on the contract or excess after the staking program ends.
-     * @dev Any wallet can call this function as it will only ever send tokens to the owner of the distributor.
-     * @param tokenAddress The address of the token to recover.
-     * @param amount The amount of the token to recover.
+     * @notice Enables the owner to recover tokens dropped onto the contract. This could be used to remove unclaimed
+     * staking rewards or recover excess LP tokens that were inadvertently dropped onto the contract. Importantly, the
+     * contract will only let the owner recover staked excess tokens above what the contract thinks it should have. i.e
+     * the owner cant use this method to steal staked tokens, only recover excess ones mistakenly sent to the contract.
+     * @param token The address of the token to skim.
      */
-    function recoverErc20(address tokenAddress, uint256 amount) external nonReentrant {
-        require(stakingTokens[tokenAddress].lastUpdateTime == 0, "Can't recover staking token");
-        require(tokenAddress != address(rewardToken), "Can't recover reward token");
-        IERC20(tokenAddress).safeTransfer(owner(), amount);
-
-        emit RecoverErc20(tokenAddress, owner(), amount, msg.sender);
+    function recoverToken(address token) external onlyOwner {
```

Comment:
This function will be bricked if we set the owner to the 0x0 address. Is this something we will never do? Also we should document this fact, or handle it